src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/DataflowReadsFilter.java

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.hellbender.dev.pipelines.bqsr;
+
+import com.google.cloud.dataflow.sdk.transforms.DoFn;
+import com.google.cloud.dataflow.sdk.transforms.PTransform;
+import com.google.cloud.dataflow.sdk.transforms.ParDo;
+import com.google.cloud.dataflow.sdk.values.PCollection;
+import htsjdk.samtools.SAMFileHeader;
+import org.broadinstitute.hellbender.engine.filters.ReadFilter;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.MutableGATKRead;
+
+import java.io.Serializable;
+
+/**
+ * A simple filter for reads on dataflow
+ */
+public final class DataflowReadsFilter extends PTransform<PCollection<MutableGATKRead>, PCollection<MutableGATKRead>> implements Serializable {
```

Comment:
also, should transforms have active, verb-like, names like 'FilterReadsTransform' or something? we need a convention and use it throughout